modin/test/test_logging.py

```diff
@@ -27,8 +27,8 @@ def __init__(self, namespace):
         self.messages = collections.defaultdict(list)
         self.namespace = namespace
 
-    def info(self, message, *args, **kw):
```

Comment:
Why changed?

Reply:
In previous impl, as log_level was a string the  logger was called as below, 

```
logger_level = getattr(logger, log_level)
logger_level(start_line)
```

now as log level is string this has been changed to call logger.log(log_level,message)